src/main/java/cz/centrum/haffner/SimpleTrainingDavid/Kafka/KafkaConsumerConfig.java

```diff
@@ -0,0 +1,47 @@
+package cz.centrum.haffner.SimpleTrainingDavid.Kafka;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// TODO: class not used, just a concept
+@EnableKafka
+@Configuration
+public class KafkaConsumerConfig {
+
+    @Bean
+    public ConsumerFactory<String, String> consumerFactory() {
```

Comment:
Doporučuju projít si tento článek:
https://www.baeldung.com/properties-with-spring
A vybrat odtud implementaci, která používá načítání hodnot definovaných v application.properties

Potom bude stačit změnit property file a nebudeš muset kompilovat celý projekt, ve kterém máš hardcoded hodnoty.

Reply:
předěláno